src/GeositeFramework/js/util/ajax.js

```diff
@@ -11,16 +11,27 @@ define(['esri/request'],
         }
 
         // Fetch data from `url` and store the result in `cache` (memoized).
-        var fetch = function(url) {
+        var fetch = function(url, format, postprocess) {
```

Comment:
Consider accepting an `options` argument here instead that we can pass directly to the request function. We could use `_.defaults` to provide some default values.